HunterPie.Native/Core/Server/Handlers/Initialize/InitializationMessageHandler.h

```diff
@@ -7,11 +7,26 @@ namespace Core::Server
 {
     namespace Models
     {
+        enum class IPCInitializationHostType : uint32_t
+        {
+            Invalid = 0,
+            MHWorld,
+            MHRise,
+        };
+
         struct RequestIPCInitializationMessage : IPCMessage
         {
+            IPCInitializationHostType hostType;
+            // For x64 word alignment.
+            uint32_t reserved1;
```

Comment:
I don't think this is needed, and also pollutes the structure definition

Reply:
I think we can remove this under the assumption that the word alignment is the same across all the apps. I'll remove it.